ruby_on_rails/configure_ci.md

```diff
@@ -13,31 +13,8 @@ and the main branches already pushed and ready to be tested.
 
 ## Rails specific configuration
 
-### Using Renuo CLI:
-
 ```
 renuo configure-semaphore
 ```
 
-The command will copy the templates to `.semaphore` folder.
-
-### Manually
-
-1. Create a folder called `.semaphore` in the project root with a file called `semaphore.yml` in it.
-
-1. Take the content of [this template](../templates/.semaphore) and replace [project-name] where
-needed.
-
-    This runs a build and a testing process for your code.
-
-1. Add a file called `.nvmrc` to the project root, where you specify the latest node version
-
-1. Head to <https://renuo.semaphoreci.com/secrets> and add a new secret with the name of your project.
-
-1. Commit the files to all three branches, push and watch the CI run.
-
-When you have your three green builds you have configured your CI properly.
-
-![semaphoreci_2](../images/semaphore_ci.png)
```

Comment:
I wouldn't remove everything, because the command doesn't cover all the points written under the title "Manually". Following points should be kept in order not to forget anything:

"1. Add a file called '.nvmrc' to the project root, where you specify the latest node version.
2. Commit the files to all three branches, push and watch the CI run.

When you have your three green builds you have configured your CI properly.

![semaphoreci_2](../images/semaphore_ci.png)"

Reply:
agreed 👍